src/methods/create.ts

```diff
@@ -56,11 +55,8 @@ export async function create<S extends PropertiesSchema>(properties: Configurati
 
 function buildIndex<S extends PropertiesSchema>(lyra: Lyra<S>, schema: S, prefix = "") {
   for (const prop of Object.keys(schema)) {
```

Comment:
The `Object.keys` method always returns an array of strings, so we can avoid the `propType` assignment and the throw error.